libraries/I2S/src/I2S.cpp

```diff
@@ -559,6 +559,14 @@ bool I2S::read32(int32_t *l, int32_t *r) {
     return true;
 }
 
+size_t I2S::read(const uint8_t *buffer, size_t size) {
```

Comment:
Same `const` thought.  POSIX `write` is non-const, only `read` is.  We're overwriting data in the ptr so it's really not const anyway (wouldn't be very useful of a read if we couldn't get new data :laughing: )